voices.json

```diff
@@ -0,0 +1 @@
+{"voices": [{"languageCodes": ["pl-PL"], "name": "pl-PL-Standard-A", "ssmlGender": "FEMALE", "naturalSampleRateHertz": 24000}, {"languageCodes": ["pl-PL"], "name": "pl-PL-Standard-B", "ssmlGender": "MALE", "naturalSampleRateHertz": 24000}, {"languageCodes": ["pl-PL"], "name": "pl-PL-Standard-C", "ssmlGender": "MALE", "naturalSampleRateHertz": 24000}, {"languageCodes": ["pl-PL"], "name": "pl-PL-Standard-E", "ssmlGender": "FEMALE", "naturalSampleRateHertz": 24000}, {"languageCodes": ["pl-PL"], "name": "pl-PL-Standard-D", "ssmlGender": "FEMALE", "naturalSampleRateHertz": 24000}, {"languageCodes": ["pl-PL"], "name": "pl-PL-Wavenet-A", "ssmlGender": "FEMALE", "naturalSampleRateHertz": 24000}, {"languageCodes": ["pl-PL"], "name": "pl-PL-Wavenet-B", "ssmlGender": "MALE", "naturalSampleRateHertz": 24000}, {"languageCodes": ["pl-PL"], "name": "pl-PL-Wavenet-C", "ssmlGender": "MALE", "naturalSampleRateHertz": 24000}, {"languageCodes": ["pl-PL"], "name": "pl-PL-Wavenet-E", "ssmlGender": "FEMALE", "naturalSampleRateHertz": 24000}, {"languageCodes": ["pl-PL"], "name": "pl-PL-Wavenet-D", "ssmlGender": "FEMALE", "naturalSampleRateHertz": 24000}, {"languageCodes": ["pl-PL"], "name": "pl-PL-Standard-A", "ssmlGender": "FEMALE", "naturalSampleRateHertz": 24000}, {"languageCodes": ["pl-PL"], "name": "pl-PL-Standard-B", "ssmlGender": "MALE", "naturalSampleRateHertz": 24000}, {"languageCodes": ["pl-PL"], "name": "pl-PL-Standard-C", "ssmlGender": "MALE", "naturalSampleRateHertz": 24000}, {"languageCodes": ["pl-PL"], "name": "pl-PL-Standard-D", "ssmlGender": "FEMALE", "naturalSampleRateHertz": 24000}, {"languageCodes": ["pl-PL"], "name": "pl-PL-Standard-E", "ssmlGender": "FEMALE", "naturalSampleRateHertz": 24000}]}
```

Comment:
How about moving this file to a `data` folder or smth like that? I will also have some configuration json files and we could store them there